prdoc/pr_4912.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Removed `pallet::getter` usage from the pallet-babe
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removed `pallet::getter`s from `pallet-babe`s storage items.
+      When accessed inside the pallet, use the syntax `StorageItem::<T>::get()`.
```

Comment:
Can you update this please?  
It just uses less FRAME macro magic now as it seems.

Reply:
nvm, its probably `minor` because of the type that is being made public. Then please ignore my other comments about this.

Reply:
Sure, thank you

Reply:
Actually I have add the above description taking this PR as reference: https://github.com/paritytech/polkadot-sdk/pull/4513

What should I add here? Can you please suggest?